### PR TITLE
Rename directories to remove helium references and ocean

### PR DESCRIPTION
**Pull request by stevededalus**
Will follow up renaming exceptions and classes in another PR.

https://app.asana.com/0/855836770921302/874855898573849/f

### CODE REVIEW

File: sdk/python/t4/api.py [hunk 1]
```diff
@@ -374,7 +374,7 @@ def config(*autoconfig_url, **config_values):
     """Set or read the Helium configuration
 
     To retrieve the current config, call directly, without arguments:
-        >>> import helium as he
+        >>> import t4 as he
         >>> he.config()
```
Comment (dimaryaz) [inline sdk/python/t4/api.py]: We should probably call it `t4` rather than `he` now?

Comment (stevededalus) [reply sdk/python/t4/api.py]: Yeah, going to do a follow up rename of all these and the Exceptions names and docs.

**Comment by akarve** [comment]
i suspect people looking for a "way to use T4 in Python" would find `api/` more easily, but i'm not religous about this so pick the one you think works best

**Comment by stevededalus** [comment]
Ok, switched to API, PTAL

**Pull request merged**